packages/safe-tools-client/app/templates/application.hbs

```diff
@@ -1,7 +1,48 @@
-{{page-title 'ScheduledPaymentsClient'}}
+{{page-title 'Cardstack Safe Tools'}}
 
-<div class="Application--safe-tools-dashboard">
+<Boxel::Dashboard @darkTheme={{true}}>
+  <:header>
+    <Boxel::OrgHeader
+      class='safe-tools-header'
+      @title='CARDSTACK'
+      @subtitle='Safe Tools'
+      @iconURL='/dashboard/cardstack-logo.svg'
+      style={{
+        css-var
+        boxel-org-header-background-color='var(--boxel-dark)'
+        boxel-org-header-logo-size='var(--boxel-sp-xl)'
+        boxel-org-header-logo-position='right'
+        boxel-org-title-title-font='300 var(--boxel-font-size-lg) var(--boxel-font-family)'
+        boxel-org-title-subtitle-font='300 var(--boxel-font-size-xs) var(--boxel-font-family)'
+        boxel-org-title-letter-spacing='var(--boxel-lsp-sm)'
+        boxel-org-title-text-transform='none'
+      }}
```

Comment:
Is there a specific reason to use the style property, instead of using class ?

Reply:
Personal preference - I'd go with styles in templates for first-class Boxel CSS variables so that it's clearer what needs to change when Boxel components change.

Reply:
I don’t really like it in templates but that’s the usual way we do it so that’s what I stuck with 🥴